src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -358,8 +364,8 @@ def register_dataset_udf(
     compressor = kwargs.pop("compressor", None)
 
     # Get either the new experimental or default reader
-    reader = kwargs.pop("reader", "production")
-    experimental_reader = True if reader == "experimental" else False
+    reader = kwargs.pop("reader", ReaderType.PRODUCTION.value)
```

Comment:
@ktsitsi how about making `reader` a name keyword argument of `ingest()` instead?

I'm doing a deep dive into our ingestion code right now and the way we have been moving things in and out of `kwargs` obfuscates the code.

Reply:
It indeed does however, the reason I did not do it in the first place is to avoid deprecating it later and breaking the API in the next releases.